Xamarin.Forms.Core/Items/ItemSizingStrategy.cs

```diff
@@ -3,6 +3,7 @@
 	public enum ItemSizingStrategy
 	{
 		MeasureAllItems,
-		MeasureFirstItem
+		MeasureFirstItem,
+		None
```

Comment:
What does `None` do?

Reply:
None says , hey don t measure the items , the Parent (CarouselView) will be setting the size of the items. 
Theres are some points on the shared ItemsVIewRenderer and controllers that take MeasureFirstItem in account, and since it s the default for the enum.